crates/matrix-sdk-ui/src/notification_client.rs

```diff
@@ -259,10 +256,21 @@ impl NotificationClient {
         match encryption_sync {
             Ok(sync) => match sync.run_fixed_iterations(2, sync_permit_guard).await {
                 Ok(()) => match room.decrypt_event(raw_event.cast_ref()).await {
-                    Ok(new_event) => {
-                        trace!("Encryption sync managed to decrypt the event.");
-                        Ok(Some(new_event))
-                    }
+                    Ok(new_event) => match new_event.kind {
+                        matrix_sdk::deserialized_responses::TimelineEventKind::UnableToDecrypt {
+                            utd_info, ..
+                        } => {
+                            trace!(
```

Comment:
Maybe here too?

Reply:
Again, the actual error will be logged at warning, and in this case it's just replicating the previous behaviour (note the `trace!("Encryption sync failed to decrypt the event: {err}");` just below).

I can change this if you feel strongly but it's not obvious to me that we should.

Reply:
No no, that's quite okay, I didn't check the lower levels to see if we print it anywhere else.